docs/references/annotations.md

```diff
@@ -129,6 +131,16 @@ Please follow the
 [Using the KongIngress resource](../guides/using-kongingress-resource.md)
 guide for details on how to use this annotation.
 
+## `configuration.konghq.com/protocol`
+
+This annotation can set a protocol � `http`, `https`, `grpc`, or `grpcs` �
+on a Service resource. The services you define will map to the [Kong Service](https://docs.konghq.com/1.4.x/admin-api/#service-object), operating on the protocol that you set.
+
+## `configuration.konghq.com/protocols`
+
+This annotation can set a pair of protocols (`http`,`https`) or (`grpc`,`grpcs`)
+on an Ingress resource. The paths in the Ingress resources you define will map to the [Kong Routes](https://docs.konghq.com/1.4.x/admin-api/#route-object), operating on the protocol that you set.
+
```

Comment:
Let's try to keep these lines under 80 columns limit.
Two changes:
1. Can you change the link to `latest` instead of  `1.4.x` so that it is automatically kept up to date?
2. Can we also add in as to what it means? For `protocol`, I'd add that: "This sets the protocol to be used for communication between Kong and the service, internally in the k8s cluster.", and similar for `protocols`, I'd say that it is the protocol to be used for communication between the external user/service and the ingress point (kong).